ACadSharp.Tests/Entities/MTextTests.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using ACadSharp.Entities;
+using CSMath;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace ACadSharp.Tests.Entities
+{
+    public class MTextTests
+    {
+
+        [Fact]
+        public void RotationDoesNotOverrideAlignmentPoint()
+        {
+            var text = new MText
+            {
+                Rotation = Math.PI
+            };
+
+            Assert.Equal(text.AlignmentPoint, XYZ.Zero);
```

Comment:
This test needs to be reviewed, `AlignmentPoint` is a unitary vector, that means that it will never be 0, the modulus will always be 1 (NOTE: this has been implemented as a point so the modulus can be higher by now)

Also, PI in radians is 180 degrees, meaning the alignment point will be (-1,0,0), right now in the test the result is (-1,1.2246467991473532E-16,0) which is "correct" but it needs to correct the almost zero to 0.